subscription-manager.spec

```diff
@@ -340,6 +340,7 @@ rm -rf %{buildroot}
 %attr(755,root,root) %dir %{_sysconfdir}/rhsm
 %attr(755,root,root) %dir %{_sysconfdir}/rhsm/facts
 %attr(644,root,root) %config(noreplace) %{_sysconfdir}/rhsm/rhsm.conf
+%config(noreplace) %attr(644,root,root) %{_sysconfdir}/rhsm/logging.conf
```

Comment:
I think we should remove the "noreplace" here (so just `%config`) because we probably do want to replace it eventually.